Localization/localization.cfg

```diff
@@ -131,135 +131,135 @@ Localization
+        precisemaneuver_keybindings_normzero            = Anular normal
+        precisemaneuver_keybindings_radinc              = Incrementar radial
+        precisemaneuver_keybindings_raddec              = Reducir radial
+        precisemaneuver_keybindings_radzero             = Anular radial
+        precisemaneuver_keybindings_timeinc             = Incrementar tiempo
+        precisemaneuver_keybindings_timedex             = Reducir tiempo
+        precisemaneuver_keybindings_circularize         = Circularizar órbita
+        precisemaneuver_keybindings_turnup              = Agregar una órbita
+        precisemaneuver_keybindings_turndown            = Reducir una órbita
+        precisemaneuver_keybindings_pageinc             = Cambiar incremento (w/Alt)
+        precisemaneuver_keybindings_pageconics          = Cambiar modo de órbita (w/Alt)
+        precisemaneuver_keybindings_encfocus            = Enfocar siguiente encuentro
+        precisemaneuver_keybindings_patchesmore         = Mostrar más parches
+        precisemaneuver_keybindings_patchesless         = Mostrar menos parches
+        precisemaneuver_keybindings_timex10             = Cambiar x10 veces
+        precisemaneuver_keybindings_nextnode            = Siguiente nodo de maniobra
+        precisemaneuver_keybindings_prevnode            = Anterior nodo de maniobra
+        precisemaneuver_keybindings_delnode             = Eliminar nodo de maniobra
+                
+        precisemaneuver_date_format                     = Año <<1>>, Día <<2>>, <<3>>:<<4>>:<<5>>
+        precisemaneuver_date_format_compact             = A<<1>> D<<2>> <<3>>:<<4>>:<<5>>
+                
+        precisemaneuver_node                            = Nodo
                 
         precisemaneuver_prefix_kilo                     = k
         precisemaneuver_prefix_mega                     = M
         precisemaneuver_prefix_giga                     = G
         precisemaneuver_meter                           = m
                 
-        precisemaneuver_pager_focus                     = Focus
-        precisemaneuver_pager_focus_tooltip             = Focus view on the maneuver node
-        precisemaneuver_pager_delete                    = Del
-        precisemaneuver_pager_delete_tooltip            = Delete current maneuver node
+        precisemaneuver_pager_focus                     = Enfocar
+        precisemaneuver_pager_focus_tooltip             = Enfocar vistar en nodo de maniobra
+        precisemaneuver_pager_delete                    = Eliminar
+        precisemaneuver_pager_delete_tooltip            = Eliminar actual nodo de maniobra
                 
-        precisemaneuver_saver_save                      = Save
-        precisemaneuver_saver_save_as                   = Save As
-        precisemaneuver_saver_delete                    = Del
+        precisemaneuver_saver_save                      = Guardar
+        precisemaneuver_saver_save_as                   = Guardar como
```

Comment:
No way I can fit that into the button. maybe use "nuevo" instead. It means more or less the same.